packages/venia-concept/package.json

```diff
@@ -73,6 +74,7 @@
     "@magento/pwa-buildpack": "~3.0.0",
     "@magento/upward-js": "~2.2.0",
     "@storybook/react": "~4.1.13",
+    "@testing-library/react": "~8.0.8",
```

Comment:
Using `@testing-library/react` for DOM testing since the test renderer that comes with react renders components all in react as JS objects. Needed something like JSDom to assert head tags. Let me know if we can achieve this without `@testing-library/react`, will be happy to change.

FYI, the API `@testing-library/react` provides is very intuitive and can be used for other tests in the future.

Reply:
Oh, I assumed @jimbo steered you in this direction. We probably want his sign off on this one.

Reply:
This is on me. Jimmy dint ask me to use this package. But if you guys have any other solution to render components on JSDOM or similar DOM implementations without a new library, let me know.

Reply:
I think this is the recommended approach, I was just surprised we didn't already have it. I'm 👍